.github/workflows/publish_charm.yaml

```diff
@@ -0,0 +1,13 @@
+name: Publish to edge
```

Comment:
Next time, can you file unrelated changes like this with another PR?

Reply:
Ok